### PR TITLE
Add Bootstrap based theming guide

### PR DESCRIPTION
**Pull request by kulczy**

### CODE REVIEW

File: docs/theming/bootstrap-theme.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+``$primary: blue;``, and then import this file into ``index.scss``.
+
+.. tip::
+
+    Variables should be overwritten before importing styles from BootstrapTheme, so the ``_variables.scss`` file should be imported at the beginning of the ``index.scss`` file.
+
+Adding new assets
+~~~~~~~~~~~~~~~~~
+
+To add a new assets to the theme, such as scripts or images, simply place them in your theme's directory and then import them into the file ``entry.js``
```
Comment (GSadee) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
To add new assets to the theme, such as scripts or images, simply place them in your theme's directory and then import them into the file ``entry.js``
```

File: docs/theming/bootstrap-theme.rst [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+Install missing BootstrapTheme dependencies
+
+.. code-block:: bash
+
+    yarn add node-sass lodash.throttle -D
+    yarn add bootstrap bootstrap.native glightbox axios form-serialize @fortawesome/fontawesome-svg-core @fortawesome/free-brands-svg-icons @fortawesome/free-regular-svg-icons @fortawesome/free-solid-svg-icons
+
+in ``theme/BootstrapChildTheme/assets`` create 2 files: ``entry.js`` and ``scss/index.scss``
+
+``entry.js`` is the main file for your theme. All files used in the theme will be imported here. First, add the files used in the BootstrapTheme and your newly created scss file
```
Comment (CoderMaggie) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
``entry.js`` is the main file for your theme. All files used in the theme will be imported here. 
First, add the files used in the BootstrapTheme and your newly created scss file
```

File: docs/theming/bootstrap-theme.rst [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+and in the ``config/packages/webpack_encore.yaml`` add:
+
+.. code-block:: yaml
+
+    webpack_encore:
+        output_path: '%kernel.project_dir%/public/build/default'
+        builds:
+            bootstrapTheme: '%kernel.project_dir%/public/bootstrap-theme'
+
+Now you can use one of the commands ``yarn encore dev``, ``yarn encore production`` or ``yarn encore dev-server`` to compile all assets. Open the page - everything should work.
```
Comment (CoderMaggie) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
Now you can use one of the commands ``yarn encore dev``, ``yarn encore production`` or ``yarn encore dev-server``
to compile all assets. Open the page - everything should work.
```

File: docs/theming/bootstrap-theme.rst [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+.. _customization:
+
+3. Customization
+^^^^^^^^^^^^^^^^
+
+Changing styles
+~~~~~~~~~~~~~~~
+
+To add new styles, create a new scss file in your theme's ``assets`` folder, and then import it into the ``index.scss``. After compilation, new styles should appear on the page.
```
Comment (CoderMaggie) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
To add new styles, create a new scss file in your theme's ``assets`` folder, and then import it into the
``index.scss``. After compilation, new styles should appear on the page.
```

File: docs/theming/bootstrap-theme.rst [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+You can also override the default styles used in BootstrapTheme by changing some variables. To do that, create a file ``_variables.scss`` in the ``assets`` folder, change e.g. primary color by typing
```
Comment (CoderMaggie) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
You can also override the default styles used in BootstrapTheme by changing some variables. To do that,
create a file ``_variables.scss`` in the ``assets`` folder, change e.g. primary color by typing
```

File: docs/theming/bootstrap-theme.rst [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+``$primary: blue;``, and then import this file into ``index.scss``.
+
+.. tip::
+
+    Variables should be overwritten before importing styles from BootstrapTheme, so the ``_variables.scss`` file should be imported at the beginning of the ``index.scss`` file.
```
Comment (CoderMaggie) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
    Variables should be overwritten before importing styles from BootstrapTheme, so the ``_variables.scss``
    file should be imported at the beginning of the ``index.scss`` file.
```

File: docs/theming/bootstrap-theme.rst [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+Adding new assets
+~~~~~~~~~~~~~~~~~
+
+To add new assets to the theme, such as scripts or images, simply place them in your theme's directory and then import them into the file ``entry.js``
```
Comment (CoderMaggie) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
To add new assets to the theme, such as scripts or images, simply place them in your theme's directory
and then import them into the file ``entry.js``
```

File: docs/theming/bootstrap-theme.rst [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+
+Overwriting templates
+~~~~~~~~~~~~~~~~~~~~~
+
+To overwrite the template, copy the selected twig file from BootstrapTheme and paste it into the same place in your theme. For example, if you want to change something in the ``layout.html.twig`` file, copy to ``themes/BootstrapChildTheme/SyliusShopBundle/views``
```
Comment (CoderMaggie) [inline docs/theming/bootstrap-theme.rst]:
```suggestion
To overwrite the template, copy the selected twig file from BootstrapTheme and paste it into the same place
in your theme. For example, if you want to change something in the ``layout.html.twig`` file,
copy it to ``themes/BootstrapChildTheme/SyliusShopBundle/views``.
```

**Comment by pamil** [comment]
Thanks, Szymon! :1st_place_medal:

**Pull request merged**